Firmware/ultralcd.cpp

```diff
@@ -3198,9 +3192,6 @@ uint8_t lcd_show_multiscreen_message_with_choices_and_wait_P(
             if (allow_timeouting && _millis() - previous_millis_cmd > LCD_TIMEOUT_TO_STATUS) {
                 return LCD_BUTTON_TIMEOUT;
             }
-            manage_heater();
-            manage_inactivity(true);
```

Comment:
Why was this removed? Is lcd_show_multiscreen_message_with_choices_and_wait_P always accompanied with some other ways of managing heaters and inactivity?

Reply:
There is a call yo delay_keep_alive above. It calls these functions. There is no need to do it twice, that just increases code size.

Reply:
:+1: